public/app/features/browse-dashboards/components/NameCell.tsx

```diff
@@ -53,6 +53,9 @@ export function NameCell({ row: { original: data }, onFolderClick, treeID }: Nam
     );
   }
 
+  // We don't link to the Shared with me pseudo-folder
+  const itemURL = item.url && !isSharedWithMe(item) ? item.url : undefined;
```

Comment:
can we do this way earlier and not even set it in the state? then this component requires no changes

like here: https://github.com/grafana/grafana/blob/main/public/app/features/browse-dashboards/api/services.ts#L39

Reply:
Ahh! That was actually my preferred approach but I couldn't see how that was linked in in browseDashboardsApi.ts! I'll give this a go.

Reply:
were you able to check if this was possible as well? 🤔

Reply:
yup - i think i got confused reading the RTKQ stuff.